pkg/plugins/optional/monitoring/v1alpha/scaffolds/internal/templates/metricsdocs.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	"path/filepath"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+)
+
+var _ machinery.Template = &MetricsdocsManifest{}
+
+// Kustomization scaffolds a file that defines the kustomization scheme for the metrics folder
+type MetricsdocsManifest struct {
+	machinery.TemplateMixin
+}
+
+// SetTemplateDefaults implements file.Template
+func (f *MetricsdocsManifest) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = filepath.Join("monitoring", "metrics", "metricsdocs.go")
+	}
+
+	f.TemplateBody = MetricsdocsTemplate
+	f.IfExistsAction = machinery.OverwriteFile
+
+	return nil
+}
+
+// nolint: lll
+const MetricsdocsTemplate = `
+package main
+
+import (
+	"fmt"
+	"sort"
+
+	""sigs.k8s.io/kubebuilder/v3/pkg/plugins/optional/monitoring/v1alpha/scaffolds/internal/templates/metrics.go"
+	""sigs.k8s.io/kubebuilder/v3/pkg/plugins/optional/monitoring/v1alpha/scaffolds/internal/templates/util.go""
+)
+
+// constant parts of the file
+const (
+	title      = "# Memcached Operator Metrics\n"
```

Comment:
@avlitman is this supposed to be here? I believe all references to the  Memcached Operator should be removed from the PR